go/arrow/array/numericbuilder.gen.go.tmpl

```diff
@@ -203,6 +203,21 @@ func (b *{{.Name}}Builder) AppendValueFromString(s string) error {
       return err
     }
     b.Append(arrow.Date64FromTime(tm))  
+  {{else if or (eq .Name "Dictionary") -}}
```

Comment:
no need for the `or` here

Reply:
removed in 720ce704e0b6fa4f6bd2f063bdae0a6b94d34b3f